spec/boot_libultra_gc.inc

```diff
@@ -17,6 +17,7 @@
     include "$(BUILD_DIR)/src/libultra/os/writebackdcache.o"
     include "$(BUILD_DIR)/src/libultra/io/vigetnextframebuf.o"
     include "$(BUILD_DIR)/src/libultra/io/pimgr.o"
+    include "$(BUILD_DIR)/src/libultra/os/kdebugserver.o"
```

Comment:
Just curious, why did you add this to gc & n64 boot segments?

Reply:
Since `__osThreadSave` was moved out of pimgr and into this file, this change applies to all versions